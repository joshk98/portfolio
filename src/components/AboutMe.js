import React from "react";
import Skills from "./Skills";
import ContactMe from "./ContactMe";

import "../styles/aboutme.css";

function AboutMe() {
  return (
    <div>
      <div className="aboutme-container">
        <img
          src="https://t4.ftcdn.net/jpg/00/97/58/97/360_F_97589769_t45CqXyzjz0KXwoBZT9PRaWGHRk5hQqQ.jpg"
          alt="profilepic"
        />
        <div className="aboutme-container-description">
          <h1>Hello!</h1>
          <p>
            My name is <strong>Josh Kuhn.</strong> I am a project manager based
            in Belfast, UK. I have a passion for web development and project
            management. I created this website to demonstrate my skills in web
            development and to share projects I am working on.
          </p>
          <p>Some more of my qualifications include:</p>
          <ul>
            <li>
              Masters degree in Business Administration & Project Management
            </li>
            <li>Completed coursework in Software Engineering</li>
            <li>Professional Scrum Master certification</li>
          </ul>
        </div>
      </div>
      <Skills />
      <ContactMe />
    </div>
  );
}

export default AboutMe;
