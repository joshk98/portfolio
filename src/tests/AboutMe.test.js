import React from "react";
import { render } from "@testing-library/react";
import AboutMe from "../components/AboutMe";

describe("AboutMe", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<AboutMe />);

    expect(asFragment()).toMatchSnapshot();
  });
});
