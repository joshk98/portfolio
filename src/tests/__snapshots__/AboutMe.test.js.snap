// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMe renders correctly 1`] = `
<DocumentFragment>
  <div>
    <div
      class="aboutme-container"
    >
      <img
        alt="profilepic"
        src="img-self.jpg"
      />
      <div
        class="aboutme-container-description"
      >
        <h1>
          Hello!
        </h1>
        <h1>
          I'm Joshua Kuhn.
        </h1>
        <p>
          I'm a project manager currently based in Belfast, UK. I have a passion for 
          <strong>
            project management
          </strong>
           and software development. I created this website to demonstrate my skills in 
          <strong>
            software development
          </strong>
           and to share projects I'm working on.
        </p>
        <p>
          Some more of my qualifications include:
        </p>
        <ul>
          <li>
            Masters degree in Business Administration & Project Management
          </li>
          <li>
            Completed coursework in Software Development
          </li>
          <li>
            Professional Scrum Master certification
          </li>
        </ul>
      </div>
    </div>
    <div
      class="skills-container"
    >
      <h1>
        Skills
      </h1>
      <div
        class="skills"
      >
        <div
          class="skills-type"
        >
          <h2>
            Front-End
          </h2>
          <div
            class="skills-images"
          >
            <div>
              <img
                alt="html5"
                src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/html5/html5-original.svg"
              />
            </div>
            <div>
              <img
                alt="css3"
                src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/css3/css3-original.svg"
              />
            </div>
            <div>
              <img
                alt="javascript"
                src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/javascript/javascript-original.svg"
              />
            </div>
            <div>
              <img
                alt="react"
                src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/react/react-original.svg"
              />
            </div>
            <div>
              <img
                alt="figma"
                src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/figma/figma-original.svg"
              />
            </div>
          </div>
        </div>
        <div
          class="skills-type"
        >
          <h2>
            Back-End
          </h2>
          <div
            class="skills-images"
          >
            <div>
              <img
                alt="nodejs"
                src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/nodejs/nodejs-original.svg"
              />
            </div>
            <div>
              <img
                alt="sequelize"
                src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/sequelize/sequelize-original.svg"
              />
            </div>
            <div>
              <img
                alt="postgresql"
                src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/postgresql/postgresql-original.svg"
              />
            </div>
            <div>
              <img
                alt="docker"
                src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/docker/docker-original.svg"
              />
            </div>
            <div>
              <img
                alt="amazonwebservices"
                src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/amazonwebservices/amazonwebservices-original.svg"
              />
            </div>
          </div>
        </div>
        <div
          class="skills-type"
        >
          <h2>
            Other
          </h2>
          <div
            class="skills-images"
          >
            <div>
              <img
                alt="bash"
                src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/bash/bash-original.svg"
              />
            </div>
            <div>
              <img
                alt="git"
                src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/git/git-original.svg"
              />
            </div>
            <div>
              <img
                alt="github"
                src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/github/github-original.svg"
              />
            </div>
            <div>
              <img
                alt="wordpress"
                src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/wordpress/wordpress-original.svg"
              />
            </div>
            <img
              alt="asana"
              src="https://cdn.worldvectorlogo.com/logos/asana-logo.svg"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      class="contactme-container"
      id="contactme-section"
    >
      <h2>
        Let's keep in touch! :)
      </h2>
      <form>
        <input
          id="name"
          name="name"
          placeholder="Your Name"
          required=""
          type="text"
          value=""
        />
        <input
          id="email"
          name="email"
          placeholder="Your Email"
          required=""
          type="email"
          value=""
        />
        <textarea
          id="message"
          name="message"
          placeholder="Message"
          required=""
        />
        <button
          type="submit"
        >
          Submit
        </button>
      </form>
      <div
        class="contactme-container-links"
      >
        <a
          href="https://linkedin.com/in/joshua-kuhn"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="linkedin"
            src="icon-linkedin.png"
          />
        </a>
        <a
          href="https://github.com/joshk98"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="github"
            src="icon-github.png"
          />
        </a>
      </div>
      <strong>
        Created by Joshua Kuhn
      </strong>
    </div>
  </div>
</DocumentFragment>
`;
